platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -35,7 +35,11 @@
 #include "servers/display_server.h"
 
 #ifdef X11_ENABLED
-#include "display_server_x11.h"
+#include "x11/display_server_x11.h"
+#endif
+
+#ifdef WAYLAND_ENABLED
+#include "wayland/display_server_wayland.h"
 #endif
```

Comment:
```suggestion
#ifdef WAYLAND_ENABLED
#include "wayland/display_server_wayland.h"
#endif

#ifdef X11_ENABLED
#include "x11/display_server_x11.h"
#endif
```

Reply:
Great suggestion, I'll implement it ASAP, thanks!

Reply:
Done!